packages/table/src/common/_variables.scss

```diff
@@ -49,3 +49,9 @@ $resize-horizontal-cursor: ns-resize !default;
 
 // Actions
 $action-cursor: pointer !default;
+
+/*
+Z-index layers
+*/
+$interactive-cell-z-index: 10;
+$region-layer-z-index: 9;
```

Comment:
1. add `!default` so these can be customized
2. make them based on `$pt-z-index-content [- 1]` ?